test/.default_config

```diff
@@ -1,3 +1,3 @@
-tests.build-image-version=0.10.0-rc1
```

Comment:
This should IMHO point to latest available image tags, it is 0.10.0-rc2 currently.

Reply:
@radtriste Wouldn't it have sense to point to `latest` tag?

Reply:
in master it would